cmd/ssh-tpm-keygen/main.go

```diff
@@ -133,6 +150,14 @@ func main() {
 	}
 	defer tpm.Close()
 
+	// Ask for owner password
+	var ownerPassword []byte
+	if askOwnerPassword {
+		ownerPassword = getOwnerPassword()
```

Comment:
`pinentry` has a terminal mode I think we can just omit this and use pinentry for all of it.

Reply:
Do you know how to invoke the terminal mode? I only found a "loopback" mode of GnuPG, but wasn't able to invoke it easily from the Go wrapper.

Reply:
Considering that there is a request for ssh-askpass in #47, and since pin entry for keys is handled the same way, I would prefer to leave it as it is for now and handle pinentry or ssh-askpass migration in a follow-up PR. Would that be ok with you?